src/main/java/org/venice/beachfront/bfapi/BfApiConfig.java

```diff
@@ -105,12 +109,17 @@
 import springfox.documentation.service.Contact;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
+import util.GeoAxisJWTUtility;
 import util.PiazzaLogger;
 
 @Configuration
 public class BfApiConfig {
 	@Value("${http.keep.alive.duration.seconds}")
 	private int httpKeepAliveDurationSeconds;
+	@Value("${GEOAXIS_JWT_CERT:}")
+	private String geoaxisJwtCertPath;
+	@Value("${jwt.enabled}")
```

Comment:
Consider adding a default value to the code if the property is missing. E.g.:

`@Value("${jwt.enabled:false}")`

Also applicable to the http keep alive.

Reply:
Default values are present in application.properties file. I feel like this is likely the more proper place for them than the code itself.